system/Router/Router.php

```diff
@@ -426,6 +426,11 @@ protected function checkRoutes(string $uri): bool
                         throw RouterException::forDynamicController($handler);
                     }
 
+                    // Checks `/` in controller name
+                    if (strpos($controller, '/') !== false) {
```

Comment:
I wasn't able to find parameters for "valid PHP class names", but would it make sense to expand this out to a regex and fail anything that didn't fit?

Reply:
`Admin/AdminGlavni` is not a valid PHP class name.
Because `/` is wrong.

Many users write like `'Admin/AdminGlavni::cam_edit_show/$1'` and CI4 causes errors.

Reply:
Right, I get that. My question was: is there any reason why not to make this check for all class names? `Admin\Admin-Glavni` is also not valid. This could be implemented as a filter and the error as "Your controller contains the following invalid characters: -"